src/Controller/Component/CrudComponent.php

```diff
@@ -547,7 +547,7 @@ public function defaults($type, $name, $config = null)
                 $this->config(sprintf('%s.%s', $type, $realName), $config);
             }
 
-            return;
+            return null;
```

Comment:
You just said it twice that this change does not change anything (thus cannot break anyones app).
I hear you regarding the functional changes, but in this case even you must admit that your concerns are not justified here.